commit d7b4cfaeeadee5e0cf0ec890dd74b3b95b020007 (refs/changes/09/109/1)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2016-07-19T05:03:43+00:00 (3 years, 3 months ago)
    
    HEAD-2308 download_metadata should retry fetch from SAPI on failure
    HEAD-2310 common RBAC setup should create "prof_attr.d" if missing
    HEAD-2311 sdc-scripts bootstrap shell library must tolerate "errexit", etc
    HEAD-2302 want support for rbac/pfexec access to zone metadata
    HEAD-2305 adding files to prof_attr.d does not do anything
    TOOLS-1455 sdc-scripts.git:sbin/postlogrotate.sh can attempt to cat log files over themselves
    HEAD-2307 HEAD-2302 needs to work on old PI (missing XML file)

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index 9cd96a3..17fc5c8 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit 9cd96a35d84c223485e14f02e7a4ebca2088ca51
+Subproject commit 17fc5c8b83623078388b00260e5ad4be4f7348ac
